docs/c2/mkdocs/mkdocs.yml

```
@@ -0,0 +1,239 @@
+site_name: Terraform Rockitcloud Provider - User Guide
```

```suggestion
site_name: Terraform Rockit Cloud Provider - User Guide
```
и в `site_description` тоже